packages/expo/package.json

```diff
@@ -70,7 +70,7 @@
     "@types/react": "*",
     "@types/react-dom": "*",
     "expo-auth-session": "^5.4.0",
-    "expo-local-authentication": "^13.5.0",
+    "expo-local-authentication": "^13.8.0",
```

Comment:
This includes breaking changes, but since is a devDep it's ok

[More info here](https://github.com/expo/expo/blob/sdk-51/packages/expo-local-authentication/CHANGELOG.md#-breaking-changes-3)